lib/pf/services/manager/pfdhcp.pm

```diff
@@ -0,0 +1,60 @@
+package pf::services::manager::pfdhcp;
+=head1 NAME
+
+pf::services::manager::pfdhcp
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::services::manager::pfdhcp
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+use pf::cluster;
+use pf::config qw(
+    %Config
+);
+use pf::util;
+
+extends 'pf::services::manager';
+
+has '+name' => ( default => sub { 'pfdhcp' } );
+
+sub isManaged {
```

Comment:
No point in having this if it just call the super method unless I'm missing something